lib/generators/hyrax/templates/mediated_deposit_workflow.json.erb

```diff
@@ -58,6 +58,12 @@
                             "to": ["approving"]
                         }
                     ]
+                }, {
+                    "name": "leave_a_comment",
```

Comment:
@mjgiarlo You read the data correctly. I can write a feature test for this particular behavior if that would be more appropriate (I shudder at UI tests for data driven behavior as its verifying a certain moment, but not verifiable over the course of the project).